CMakeLists.txt

```diff
@@ -49,6 +53,16 @@ option(ENABLE_SANITIZE "Sanitize tool" OFF)
 option(ENABLE_SECURITY "Security Ford protocol protection" ON)
 option(ENABLE_HMI_PTU_DECRYPTION "Policy table update parsed by hmi" ON)
 option(ENABLE_EXTENDED_POLICY "Turns extended flow which requires embedded system interaction" ON)
+option(USE_COTIRE "Use Cotire to speed up build (currently only for commands tests)" ON)
```

Comment:
While testing this branch, it was found that there is a compiler error which occurs when both the `USE_COTIRE` and `BUILD_TESTS` options are `ON`. This needs to be fixed before the release is merged.